pandasaurus_cxg/anndata_enricher.py

```diff
@@ -102,3 +115,19 @@ def set_enricher_property_list(self, property_list: List[str]):
             property_list (List[str]): The list of properties to include in the enrichment analysis.
         """
         self.__enricher = Query(self.__seed_list, property_list)
+
+    def is_invalid_slim_list(self, slim_list):
```

Comment:
The name `is_invalid_slim_list` makes me expect this function to return a boolean value. Should we rename it to  `validate_slim_list` ?